### DISCUSSION TITLE
Applying a force BC

### DISCUSSION THREAD

**Discussion by ln53** [idaholab/moose#24561] [Q&A Modules: Solid mechanics] [answered]
Hi,

I'm making some basic linear elastic models (both shell and solid) and would like to apply a downward force to one end - similar to [this question](https://github.com/idaholab/moose/discussions/23377). However I can't find any documentation for applying a Force BC.

Several places mention using a NeumannBC (answer linked above for example), however I can't find much discussion around how this equates to force. And from what I can see the Pressure BC has had the option to specify a component removed.

- As the NeumannBC is integrated over a surface, I am assuming any Force I specify will actually have to be specified per unit area (i.e. as a stress)? With shell elements is this per unit length, or should I calculate an area using the specified shell thickness?
- In my understanding, a NeumannBC prescribes the derivative of a variable normal to a boundary. In my cases, this is the derivative (wrt y direction) of the z displacement. I think I've convinced myself spatial derivative of displacement is equivalent to strain (i.e. disp / length) - so I'm not quite sure how specifying a NeumannBC can specify stress or force directly?

Thanks!

**Comment by GiudGiud** [comment]
Hello

1) yes for the first question. @jain651 will know for the second part

2) There's a derivation here for the Jacobian, it should show the relation between force and displacements : https://mooseframework.inl.gov/source/bcs/Pressure.html

Guillaume

**Reply by ln53** [reply]
Thanks for the reply. Just to be explicit (assuming SI units):
For a face with a normal in the y direction, with an area of 1m^2, a NeumannBC with the value of -1 applied to `disp_z` will produce a total force of 1N downward on that face?
(And a similar face with an area of 0.1m^2 would require a NeumannBC with a value of -10 to produce the same 1N force?)

**Reply by ln53** [reply]
@jain651 do you know if I should account for the shell thickness (specified in ADComputeFiniteShellStrain) when I am calculating the NeumannBC value to use? (i.e. if I should divide my desired force by just side length OR side length * shell thickness)
Thanks

**Reply by jain651** [reply]
@neuphris would you please try to answer this. I am a bit busy nowadays.

**Reply by GiudGiud** [reply]
@neuphris

**Reply by ln53** [reply, marked as answer]
To conclude this: I did some basic beam bending test cases, and it seems for shells a NeumannBC is equivalent to force applied **per unit length** of the sideset. The "thickness" of the shell elements **isn't taken into account**.

So to apply a force _F_ over a side of length _L_, use a NeumannBC with a value of _F / L_.
